api/src/test/java/org/openmrs/module/idgen/service/IdentifierSourceServiceTest.java

```diff
@@ -241,7 +268,8 @@ public void getAutoGenerationOptionByPatientIdentifierAndLocation_shouldReturnNu
     }
 
     @Test(expected = NonUniqueResultException.class)
-    public void getAutoGenerationOptionByPatientIdentifierAndLocation_shouldFailWhenTryingToFetchOptionByJustPatientIdentifierIfConfiguredByLocation() {
+    public void getAutoGenerationOptionByPatientIdentifierAndLocation_shouldFailWhenTryingToFetchOptionByJustPatientIdentifierIfConfiguredByLocation
+            () {
         PatientIdentifierType patientIdentifierType = patientService.getPatientIdentifierType(2);

```

Comment:
This doesn't seem like an improvement in readability, regardless of what the Openmrs Formatter says :)